api/src/pcapi/routes/backoffice/templates/operations/details.html

```diff
@@ -0,0 +1,146 @@
+                            <li class="dropdown-item p-0">
+                              <form action="{{ url_for('backoffice_web.operations.validate_response', special_event_id=special_event.id, response_id=response.id) }}"
+                                    name="{{ url_for('backoffice_web.operations.validate_response', special_event_id=special_event.id, response_id=response.id) | action_to_name }}"
+                                    method="post">
+                                {{ csrf_token }}
+                                <button type="submit"
+                                        class="btn btn-sm d-block w-100 text-start px-3">Retenir la candidature</button>
+                              </form>
+                            </li>
+                          {% endif %}
+                          {% if response.status != response.status.PRESELECTED %}
+                            <li class="dropdown-item p-0">
+                              <form action="{{ url_for('backoffice_web.operations.preselect_response', special_event_id=special_event.id, response_id=response.id) }}"
+                                    name="{{ url_for('backoffice_web.operations.preselect_response', special_event_id=special_event.id, response_id=response.id) | action_to_name }}"
+                                    method="post">
+                                {{ csrf_token }}
+                                <button type="submit"
+                                        class="btn btn-sm d-block w-100 text-start px-3">Préselectionner la candidature</button>
+                              </form>
+                            </li>
+                          {% endif %}
+                          {% if response.status != response.status.REJECTED %}
+                            <li class="dropdown-item p-0">
+                              <form action="{{ url_for('backoffice_web.operations.reject_response', special_event_id=special_event.id, response_id=response.id) }}"
+                                    name="{{ url_for('backoffice_web.operations.reject_response', special_event_id=special_event.id, response_id=response.id) | action_to_name }}"
+                                    method="post">
+                                {{ csrf_token }}
+                                <button type="submit"
+                                        class="btn btn-sm d-block w-100 text-start px-3">Rejeter la candidature</button>
+                              </form>
+                            </li>
+                          {% endif %}
+                        </ul>
+                      </div>
+                    </td>
+                    {% if response.user %}
+                      <td>{{ links.build_public_user_name_to_details_link(response.user) }}</td>
+                      <td>{{ response.user.email }}</td>
+                      <td>{{ response.phoneNumber if response.phoneNumber else response.user.phoneNumber | empty_string_if_null }}</td>
```

Comment:
Détail : ça peut s'écrire tout simplement avec un `or` : 
```suggestion
                      <td>{{ (response.phoneNumber or response.user.phoneNumber) | empty_string_if_null }}</td>
```